_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -0,0 +1,483 @@
+---
+id: ffbc54
+name: No keyboard shortcuts use only printable characters
+rule_type: atomic
+description: |
+  This rule checks that if keyboard shortcuts are implemented using only printable characters, then there is a mechanism to disable the shortcut, or to remap the shortcut to use one or more non-printable characters keys, or the shortcut for a user interface component is only available when that component has focus.
+accessibility_requirements:
+  wcag21:2.1.4: # Character Key Shortcuts (A)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+acknowledgments:
+  authors:
+    - João Vicente
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [keyboard event][] for which all is true: 
+- the event's attribute `key` is a [printable character][] key; and
```

Comment:
Can `key` not be a printable character?
Typically because `ctrl` or `shift` are going to be passed as modifiers, not as `key`.
Similarly, the inapplicable examples with "non-printable character" should be checked to see whether it's really going to be a non-printable character, or requiring `getModifierKey` to be true.

Reply:
It can. Example: escape (there are more)

Reply:
We should have an inapplicable example with this.

Which by the way shows that Inapplicable Example 1 is not really correct (at least in its description). The event can make change if `key` is printable, as long as `getModifierState("Control")` returns `true`…

Reply:
Agree in both accounts. I've updated Example 1 to use the Escape key, irrespectively of the modifier.